templates/zabbix-agent2-systemd.init.erb

```diff
@@ -0,0 +1,16 @@
+# THIS FILE IS MANAGED BY PUPPET
```

Comment:
for the new agent, do we need to duplicate the systemd unit file? isn't there one in the package we can use?

Reply:
Do you mean that we should consider not managing the systemd unit file? If yes, then I agree. :)

Reply:
yes. if there is one we should use it. If it has any flaws we can use a drop_in to overwrite the specific settings.

Reply:
This should be fixed. Please let me know if my solution is good enough :)